apm-sniffer/apm-agent-core/src/test/java/org/apache/skywalking/apm/agent/core/plugin/witness/WitnessTest.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.skywalking.apm.agent.core.plugin.witness;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+import net.bytebuddy.matcher.ElementMatchers;
+import org.apache.skywalking.apm.agent.core.plugin.WitnessFinder;
+import org.apache.skywalking.apm.agent.core.plugin.WitnessMethod;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * witness test
+ */
+public class WitnessTest {
+
+    private String className = "org.apache.skywalking.apm.agent.core.plugin.witness.WitnessTest";
+
+    @Test
+    public void testWitnessClass() {
+        Assert.assertTrue(WitnessFinder.exist(className, this.getClass().getClassLoader()));
+    }
+
+    @Test
+    public void testWitnessMethod() {
```

Comment:
Please keep the previous version too. (name match only). It is better to have 2 test cases.